Use MaxAge as an integer value instead of metav1.Duration (#1074)
kyma-project/api-gateway: docs/user/custom-resources/apirule/04-60-apirule-v1beta2-custom-resource.md

| **corsPolicy.allowCredentials**                  |  **NO**   | Specifies whether credentials are allowed in the **Access-Control-Allow-Credentials** CORS header.                                                                                                                                                                                                                                           |                                                                                                       |
| **corsPolicy.exposeHeaders**                     |  **NO**   | Specifies headers exposed with the **Access-Control-Expose-Headers** CORS header.                                                                                                                                                                                                                                                            |                                                                                                       |

> | **corsPolicy.maxAge**                            |  **NO**   | Specifies the maximum age of CORS policy cache. The value is provided in the **Access-Control-Max-Age** CORS header. Specified as integer value of seconds.                                                                                                                                                                                  |                                                                                                       |

Reviewer: ```suggestion
| **corsPolicy.maxAge**                            |  **NO**   | Specifies the maximum age of the CORS policy cache provided in the **Access-Control-Max-Age** CORS header. The value must be an integer expressed in seconds.                                                                                                                                                      |                                                                                                       |
```

> When you fetch an existing APIRule CR, the system adds the **status** section which describes the status of the

Reviewer 2: This is not correct, since it mentions Virtual Service, we should just remove it.

VirtualService created for this CR. The following table lists the fields of the **status** section.

| Field                  | Description                                                                                                                       |
|:-----------------------|:----------------------------------------------------------------------------------------------------------------------------------|
| **status.state**       | Defines the reconciliation state of the APIRule. The possible states are `Ready`, `Warning`, `Error`, `Processing` or `Deleting`. |
| **status.description** | Detailed description of **status.state**.                                                                                         |

> **Status codes:**

Reviewer 2: I think we can delete this status codes section
